localization/english/wc_traits_l_english.yml

```
@@ -29,6 +29,9 @@
  trait_creature_high_elf:0 "High Elf"
  trait_creature_high_elf_desc:0 "The high elves, or Quel'dorei, are the absolute opposite of their cousins: the night elves. The Quel'dorei are the descendants of those who served Azshara at the time of the first Well of Eternity. The high elves used magic for generations, absorbing it with their mothers' milk. Even among those lacking a natural talent for magic, high elves cannot live or even think clearly if they do not receive their necessary mana requirement on a regular basis."
  trait_creature_high_elf_character_desc:0 "$trait_creature_high_elf_desc$"
+ trait_creature_void_elf:0 "Void Elf"
+ trait_creature_void_elf_desc:0 "Void elves are a sect of former blood elves that infused themselves with the powers of the Void for the sake of defending Azeroth, all the while resisting the Void's call to madness. In Thalassian, they are called #italic ren'dorei#!, which means "children of the Void"."
```

There was talking in the dev chat. I feel it resolves everything.
```suggestion
 trait_creature_void_elf_desc:0 "Void elves are a sect of former elves that infused themselves with the powers of the Void for the sake of defending Azeroth, all the while resisting the Void's call to madness. In Thalassian, they are called #italic ren'dorei#!, which means "children of the Void"."
```